.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This file is still in the commit because it needs to either be backed out or deleted.